structure/stack/stackarray.go

```diff
@@ -9,31 +9,45 @@
 
 package stack
 
-var stackArray []any
+type StackArray struct {
```

Comment:
Don't repeat the package name in an exported identifier. `stack.StackArray` has stack twice. `stack.Array` is shorter and gives the same idea.

Reply:
Okay, I'll change stackArray to Array